.github/workflows/observability-charm-tests.yaml

```
@@ -16,15 +16,7 @@ jobs:
       matrix:
         include:
           - charm-repo: canonical/alertmanager-k8s-operator
-            commit: 7518ec8343941ca91248614fd8f2d50fdd9e068c  # rev135 2024-09-06T12:10:02Z
-          # Waiting for an upstream PR:
-          # https://github.com/canonical/prometheus-k8s-operator/pull/639
-          #- charm-repo: canonical/prometheus-k8s-operator
-          #  commit: 7518ec8343941ca91248614fd8f2d50fdd9e068c  # rev135 2024-09-06T12:10:02Z
-          # Waiting for an upstream PR:
-          # https://github.com/canonical/grafana-k8s-operator/pull/347
-          #- charm-repo: canonical/grafana-k8s-operator
-          #  commit: 1bd9e7fd96a9f5a91443e5932667c58a92c1ce5e  # 2024-09-03T14:57:42Z
+            commit: fe233592f9324a6cf4aa9dbc71577ee170cf9eb3  # rev138 2024-10-18T12:11:02Z
```

```suggestion
          fe233592f9324a6cf4aa9dbc71577ee170cf9eb3  # rev138 2024-10-18T12:11:02Z
          # Waiting for an upstream PR:
          # https://github.com/canonical/prometheus-k8s-operator/pull/639
          #- charm-repo: canonical/prometheus-k8s-operator
          #  commit: 7518ec8343941ca91248614fd8f2d50fdd9e068c  # rev135 2024-09-06T12:10:02Z
          - charm-repo: canonical/grafana-k8s-operator
            commit: 07a52a65f272e115ab786f30cdb3b53be6b34bb7
```

Looks like grafana-k8s is fixed, but promtheus-k8s is still not.